src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeAction.cs

```diff
@@ -101,6 +102,22 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
                 return formattedDocument;
             }
 
+            private async Task<INamedTypeSymbol> GetConstructedTypeToImplementAsync(CancellationToken cancellationToken)
+            {
+                if (!_implementIEquatable) return null;
```

Comment:
📝 Can you add this to #31140? Technically it's not allowed by our code style policy, but since the build allowed it I see it as not important enough to block a pull request.

Reply:
I read https://github.com/dotnet/roslyn/blob/master/CONTRIBUTING.md#coding-style and the linked style guidelines and can't figure out what the rule is that I should add to #31140.

Is this acceptable? (rule would then be that embedded statements must start on a new line)

```cs
if (!_implementIEquatable)
    return null;
```

Or is the rule that braces are required?

Reply:
```
if (!_implementIEquatable)
    return null;
```

This form is permitted now.

Reply:
Thanks!

Reply:
> what the rule is that I should add

The rule is 17.a

Reply:
:+1: How did I miss that?